packages/core/quri_parts/core/state/state_vector_parametric.py

```diff
@@ -53,6 +53,18 @@ def __repr__(self) -> str:
     def qubit_count(self) -> int:
         return self._n_qubits
 
+    def with_primitive_circuit(self) -> "ParametricQuantumStateVector":
+        """Returns a new state with the primitive circuit applied. The
+        primitive circuit means that each gate in the circuit has an
+        independent parameter even if those in the original circuit depend on
+        the same parameters.
+
+        The original state is not changed.
+        """
```

Comment:
Probably this docstring needs to be updated in the same way as docstring of `ParametricCircuitQuantumState.with_primitive_circuit()`.

Reply:
fixed